pkg/broker/provision_job.go

```diff
@@ -59,8 +56,8 @@ func (p *ProvisionJob) Run(token string, msgBuffer chan<- WorkMsg) {
 		return
 	}
 
-	p.log.Info("Destroying APB sandbox...")
-	sm.DestroyApbSandbox(podName, p.serviceInstance.Context.Namespace)
```

Comment:
I'm not 100% sure why we moved the serviceaccountmanager into the calls out from the job. Is it simply to avoid duplication?